diff --git a/node_modules/@a2ui/lit/dist/src/0.8/ui/checkbox.js b/node_modules/@a2ui/lit/dist/src/0.8/ui/checkbox.js
index 4d64d80..38d5a77 100644
--- a/node_modules/@a2ui/lit/dist/src/0.8/ui/checkbox.js
+++ b/node_modules/@a2ui/lit/dist/src/0.8/ui/checkbox.js
@@ -135,11 +135,11 @@ let Checkbox = (() => {
                 if (!(evt.target instanceof HTMLInputElement)) {
                     return;
                 }
-                this.#setBoundValue(evt.target.value);
+                this.#setBoundValue(evt.target.checked);
             }}
         id="data"
         type="checkbox"
-        .value=${value}
+        .checked=${value}
       />
       <label class=${classMap(this.theme.components.CheckBox.label)} for="data"
         >${this.label?.literalString}</label
